sda-commons-client-jersey-wiremock-testing/src/test/java/org/sdase/commons/client/jersey/wiremock/testing/DeprecatedWireMockClassExtensionTest.java

```diff
@@ -0,0 +1,68 @@
+package org.sdase.commons.client.jersey.wiremock.testing;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.get;
+import static com.github.tomakehurst.wiremock.client.WireMock.notMatching;
+import static com.github.tomakehurst.wiremock.client.WireMock.ok;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.InputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.net.URLConnection;
+import java.nio.charset.StandardCharsets;
+import org.apache.commons.io.IOUtils;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+@Deprecated(forRemoval = true)
+class DeprecatedWireMockClassExtensionTest {
```

Comment:
The plan was to remove the deprecated Wiremock extension for v6. We should not include it in this PR, should we?

Reply:
You're right.

removed the deprecated test classes